generators/pattern.js

```diff
@@ -0,0 +1,130 @@
+const { writeFileSync, readFileSync } = require('fs');
+const { join } = require('path');
+const prompts = require('prompts');
+
+const getPatternScript = ({ patternSlug, themeSlug, description, categories }) => `<?php
+/**
+ * Title: ${description}
+ * Slug: ${themeSlug}/${patternSlug}
+ * Categories: ${categories.join(', ')}
+ */
+?>
+<!-- wp:paragraph -->
```

Comment:
I've seen these types of comments before, what is the purpose of them? Do they define the block type that WordPress uses?

Reply:
The comments are primarily for the WP block editor, and they tell WordPress which blocks are used and what additional settings are used, if any. More complicated blocks would have some JSON output in the comments for those additional settings, and if you exclude the comments, WordPress will either treat them as raw HTML and show the classic editor or show one of those invalid block content warnings.